composer.json

```diff
@@ -10,5 +10,10 @@
 		"psr-4": {
 			"BeechIt\\DefaultUploadFolder\\": "Classes/"
 		}
+	},
+	"extra": {
+		"typo3/cms": {
+			"extension-key": "default-upload-folder"
```

Comment:
This should be the extension key. so with a `_` instead of `-`

Reply:
https://github.com/beechit/default_upload_folder/pull/25/commits/6b764d16ed0feb93ed2f8ee18387f061e32b3f96